### PR TITLE
Add Washington Post paywall bypass

### PR DESCRIPTION
**Pull request by hotcap**
Hey, here's a PR for WaPo paywalls - also added support for multiple blocked JS patterns per site. Let me know if there's anything I can do to make a merge simpler for ya

### CODE REVIEW

File: src/bg/background.js [hunk 1]
```diff
@@ -17,29 +17,49 @@ var newHeader = {
 
 // sites that we want to access
 var sites = {
+	washingtonpost: {
+		js: [
+			"*://*.washingtonpost.com/*pwapi/*.js*", // This one causes paywall/ad-wall lightbox for every article
+			"*://*.washingtonpost.com/*drawbridge/drawbridge.js?_*" // This one causes paywall/ad-wall lightbox sometimes with Adblock Plus enabled
```
Comment (njuljsong) [inline src/bg/background.js]: Just curious, what do you mean by "sometimes"?

**Comment by njuljsong** [comment]
Thanks for adding Washingtonpost! I will merge this PR and do some minor update and format cleanup.

**Comment by hotcap** [comment]
Thanks for the merge! Don't know if you have a TODO list of sites but I'll try to add some more major outlets later.

**Comment by njuljsong** [comment]
@hotcap , I don't have a todo list to expand this extension to cover more sites. I prefer to keep it small. You can always fork this repo and add more sites.

File: src/bg/background.js (hunk 1 above)
Comment (hotcap) [reply src/bg/background.js]: Oh hey, didn't notice this question. I was referring to some inconsistency in which of WaPo's 2 paywalls popped up with ABP enabled, which I think may depend on which part of the site you're browsing. Compare for example https://www.washingtonpost.com/powerpost/trump-to-extend-march-5-deadline-to-end-daca-protections-if-congress-doesnt-act-gop-senator-says/2017/10/12/b64d2984-afba-11e7-be94-fabb0f1e9ffb_story.html where both paywalls seem to come up, versus https://www.washingtonpost.com/news/parenting/wp/2017/10/11/a-family-left-a-toddler-in-a-corn-maze-and-didnt-realize-it-until-the-next-day-police-say/ where I only see the left-hand paywall pop up.

**Pull request merged**